__tests__/index.test.js

```diff
@@ -366,3 +365,22 @@ describe("DELETE:/api/comments/:comment_id", () => {
       });
   });
 });
+
+describe("GET: /api/users", () => {
+  test("200: sends an array of users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.users.length).toBeGreaterThan(0);
+        response.body.users.forEach((user) => {
+          expect(user).toHaveProperty("username");
+          expect(user).toHaveProperty("name");
+          expect(user).toHaveProperty("avatar_url");
+        });
+      });
+  });
+  test("404: endpoint that does not exist", () => {
+    return request(app).get("/api/unknownendpoint").expect(404);
+  });
```

Comment:
This test shouldn't be in this describe block as it doesn't apply just to the /api/users endpoint. It's server-wide and can be in it's own describe block.